lib/ast-link-to-transform.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+const reservedProps = ['@route', '@model', '@models', '@query'];
+const supportedHTMLAttributes = ['id', 'class', 'rel', 'tabindex', 'title', 'target', 'role'];
+const attributeToPropertyMap = {
+  role: 'ariaRole',
+};
+
+class AngleBracketLinkToPolyfill {
+  transform(ast) {
+    let b = this.syntax.builders;
+
+    // in order to debug in https://https://astexplorer.net/#/gist/0590eb883edfcd163b183514df4cc717
+    // **** copy from here ****
+    function transformAttributeValue(attributeValue) {
+      switch (attributeValue.type) {
+        case 'TextNode':
+          return b.string(attributeValue.chars);
+        case 'MustacheStatement':
+          return b.path(attributeValue.path);
+      }
+    }
+
+    let visitor = {
+      ElementNode(node) {
+        if (node.tag.toLowerCase() === 'linkto') {
+          let { children, blockParams, attributes } = node;
+          let params = [];
+          let helperParams = [];
+
+          let route = attributes.find(({ name }) => name === '@route');
+          let model = attributes.find(({ name }) => name === '@model');
+          let models = attributes.find(({ name }) => name === '@models');
```

Comment:
We should probably put an assertion here that either `model` or `models` is passed, but not both, right? (as specified in the RFC)

Anything else we should cover? (an empty `route` is valid?)

Reply:
Ya, empty route is fine. Assertion for both model and models is good.